components/docs/DocSidebar.tsx

```
@@ -0,0 +1,51 @@
+
+  const renderDocItem = (item: DocItem, level = 0) => {
+    const isActive = pathname === item.path;
+    const hasChildren = item.children && item.children.length > 0;
+
+    // Styl głównej kategorii
+    if (hasChildren) {
+      return (
+        <div key={item.path} className={level === 0 ? "mb-3" : ""}>
+          <div className={`font-extrabold text-base mb-1 ${level > 0 ? "pl-4" : ""} text-gray-900 dark:text-white`}>{item.title}</div>
+          <div className="space-y-1">
+            {item.children && item.children.map((child) => renderDocItem(child, level + 1))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)